Assets/Python/Contrib/ReminderEventManager.py

```
@@ -23,293 +26,391 @@
+                    last_parts = []
+                    for text in g_turnReminderTextsList:
+                        if "\n" in text:
+                            idx = text.rfind("\n")
+                            last_parts.append(text[idx + 1:])
+                        else:
+                            last_parts.append(text)
+
+                    last_line = "".join(last_parts)
+                    width_limit = SR.x - SR.x / 7
+
+                    if CyInterface().determineWidth(last_line + reminderText) > width_limit:
```

Ruff (0.12.2)

`CyInterface` may be undefined, or defined from star imports

(F405)